modules/system/lang/th/validation.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+return [
```

Comment:
Since this file is entirely untranslated it should be removed.

Reply:
Even though it is untranslated, if we left out the file, will there be any problem if the find is not found in th folder?

Reply:
There shouldn't be, everything falls back to the default language (english), but feel free to test it.